.gitignore

```diff
@@ -0,0 +1 @@
+*.img
```

Comment:
This is interesting, is atom creating this system folder :thinking: 
I'm not familiar with it but could be a Windows thing